packages/peregrine/lib/talons/CheckoutPage/CheckoutError.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Other checkout place order mutation errors can include
+ * the following list:
+ *
+ * https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html#errors
+ */
+
+const paymentErrorMessage =
+    'Unable to place order: Transaction has been declined. Please try again later.';
+
+const removeGQLTag = rawMessage => rawMessage.replace(/GraphQL error:/, '');
+
+class CheckoutError extends Error {
+    constructor(gqlError, ...params) {
+        super(params);
+        this.name = 'CheckoutError';
+        this.message = removeGQLTag(gqlError.message);
+        this.error = gqlError;
+    }
+
+    isPaymentInfoError = () => {
```

Comment:
Are you using this function? I couldn't find reference. Also I see that the below name `paymentHasExpired` doesn't exist. I think you mean `hasPaymentExpired`?

Reply:
I don't see any `paymentHasExpired` anywhere. Sorry I didn't get you, buddy.

Reply:
You found it ;)